build/source/driver/multi_driver.f90

```diff
@@ -369,69 +525,88 @@ real(dp),parameter        :: safety = 0.01_dp               ! safety factor to e
 do istep=1,numtim
 
  ! set print flag
- globalPrintFlag=.false.
+ globalPrintFlag=.true.
 
  ! read a line of forcing data (if not already opened, open file, and get to the correct place)
- ! NOTE: only read data once: if same data used for multiple HRUs, data is copied across
  do iHRU=1,nHRU  ! loop through HRUs
-  ! assign pointers to HRUs
-  time_data => time_hru(iHRU)
-  forc_data => forc_hru(iHRU)
-  ! read forcing data
-  call read_force(istep,iHRU,err,message); call handle_err(err,message)
+  if(forcFileInfo(iHRU)%ixFirstHRU > 0)then
```

Comment:
I don't follow - comment would be helpful - are you simply reusing the forcing data if you set `ixFirstHRU` to a value greater than 0? If so that is rather cryptic.